File: MyTestProject/Delphi/TestUnit.pas
unit TestUnit;

interface

uses
  System.SysUtils, System.Types, System.UITypes, System.Classes, System.Variants,
  FMX.Types, FMX.Controls, FMX.Forms, FMX.Graphics, FMX.Dialogs, FMX.StdCtrls,
  FMX.Layouts, FMX.ListBox, FMX.Edit, FMX.Controls.Presentation;

type
  TTestForm = class(TForm)
    Label1: TLabel;
    taskEdit: TEdit;
    listBox: TListBox;
    btnAdd: TButton;
    bthDelete: TButton;
    procedure btnAddClick(Sender: TObject);
    procedure bthDeleteClick(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  TestForm: TTestForm;

implementation

{$R *.fmx}

procedure TTestForm.bthDeleteClick(Sender: TObject);
begin
//
end;

procedure TTestForm.btnAddClick(Sender: TObject);
begin
//
end;

end.

File: MyTestProject/Python/TestUnit.py
import os
from delphifmx import *


class TestForm(Form):

    def __init__(self, owner):
        self.taskEdit = None
        self.Label1 = None
        self.btnAdd = None
        self.bthDelete = None
        self.listBox = None
        self.LoadProps(os.path.join(os.path.dirname(
            os.path.abspath(__file__)), "TestUnit.pyfmx"))

    def btnAddClick(self, Sender):
        if self.taskEdit.Text != '':
            self.listBox.Items.Add(self.taskEdit.Text)
            self.taskEdit.Text = ''

    def bthDeleteClick(self, Sender):
        index = self.listBox.ItemIndex
        if index >= 0:
            self.listBox.Items.Delete(index)
